service/src/main/kotlin/fi/espoo/evaka/emailclient/EvakaEmailMessageProvider.kt

```diff
@@ -254,6 +254,24 @@ $unsubscribeEn
         )
     }
 
+    override fun missingHolidayReservationsNotification(language: Language): EmailContent {
```

Comment:
Hmm, this always returns all three languages. Is it just for possible customization's sake that we even fetch the language?

Reply:
To be investigated separately